test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13572,11 +13572,26 @@ def add_checkpoint_abort_script(self, dirname=None, body=None,
         self.du.chmod(hostname=self.hostname, path=chk_file, mode=0o700)
         self.du.chown(hostname=self.hostname, path=chk_file, runas=ROOT_USER,
                       uid=0, gid=0)
-        c = {'$action': 'checkpoint_abort ' +
+        c = {'$action checkpoint_abort':
              str(abort_time) + ' !' + chk_file + ' %sid'}
         self.add_config(c)
         return chk_file
 
+    def add_restart_script(self, dirname=None, body=None,
+                           abort_time=30):
+        """
+        Add restart script in the mom config.
+        returns: a temp file for restart script
+        """
```

Comment:
Why are we adding a method that never gets called?

Reply:
Hi @mike0042 . I plan to use this method in an upcoming pull request. This bug is blocking another bug fix of mine, which has a PTL test case that needs to add both a checkpoint abort script and a restart script.

Reply:
In other words the motivation for fixing this bug is to add this method, so I am including it in this here.